index.Rmd

```diff
@@ -17,16 +17,26 @@ favicon: "favicon/favicon.ico"
 
 # Preface {-}
 
-The purpose of this documentation is to provide content and technical guidelines for authors and editors of blog posts and tech notes for the rOpenSci website.
+The purpose of this documentation is to provide guidelines for authors and editors of blog posts and tech notes for the rOpenSci website.
 We created it after reviewing and publishing over 100 posts by community members, in hopes of addressing many frequently asked questions, and frequently given suggestions. 
 
-The volume of information may be overwhelming at first. Some of the guidance indicates things we require that you do (such as creating an author file with your metadata), while most of the guidance is information to help you (such as adding an image to your post).
-We aim to make this less daunting as we iterate. 
+The volume of information may be overwhelming at first. 
+Some of the guidance indicates things we require that you do (such as creating an author file with your metadata), while most of the guidance is information to help you (such as how to add an image to your post).
 
-This is a work in progress, in which you may stumble upon our own prompts to add more information.
-If you find any errors or things that need clarification, we would be grateful if you opened an issue or pull request in the [repository for this book](https://github.com/ropensci-org/blog-guidance).
+**What's in the book and what's required reading?**
 
-_We are thankful for the feedback and contributions by [Steffi LaZerte](https://github.com/steffilazerte)._
+- Chapter 1 [Content Guidelines](#content) coaches you in thinking through what you might say in your post.
```

Comment:
not required?

Reply:
Do you mean listing all chapters? or just ch 1 not required?

Reply:
I thought this chapter was required reading too, but only the two following chapters get the "required" alert?

Reply:
ah I see what you mean now.

imo author should read it. Was trying to go with Rich's suggestion and think critically about what is minimally required. 🤔

Reply:
I'm going to leave as is and see how things go.
If I make Ch1 required, then the bulk of the book is "required" so not so good